test/integration/switch_int_test.go

```diff
@@ -0,0 +1,111 @@
+package integration
```

Comment:
Please also add a test for tags if that's not already covered by the branch or commitID tests.

Reply:
See my comment above re: tags